model/prelude/errors.sail

```diff
@@ -10,7 +10,8 @@
 
 union exception = {
   Error_not_implemented : string,
-  Error_internal_error  : unit
+  Error_internal_error  : unit,
+  Error_reserved_behavior : string,
```

Comment:
This needs to be rebased now that [#1403](https://github.com/riscv/sail-riscv/pull/1403) was merged and this variant and the reserved_behavior function below are already present.